.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,17 @@
 Please:
 
-- [ ] Make the PR atomic. (Fix one issue at a time.) Multiple relevant issues that must be fixed together? Make atomic commits so we can easily review each issue.
-
+- [ ] Make the pull requests (PRs) atomic. (Fix one issue at a time.) Multiple relevant issues that must be fixed together? Make atomic commits so we can easily review each issue.
 - [ ] Provide a concise title so we can easily copy it to the release note.
-
   - Use imperative mood and present tense.
-  - Mention relevant issues. (e.g., `Fix #1` / `Fix part of #1`)
-
+  - Mention relevant issues. (e.g., `Fixes #1` / `Fixes part of #1`)
```

Comment:
Why fixes?  We say imperative mood the line above.

Reply:
The changes in a pull request fixes some issue. The imperative mood makes sense to say what the developer did (e.g. "remove unused function"). I feel like I always prefer to say "fixes #1" because I can't just command that the issue is fixed and instead need to make a change to indirectly fix it.

Reply:
If we said "close #1", I would agree that the imperative voice makes sense.

Reply:
or should we do past tense lol https://www.danclarke.com/git-tense

I think imperative is nice that's it's the shortest though.

Reply:
Let's stay with imperative as it's shorter.